### DISCUSSION TITLE
能不能加个选项：连接成功后自动打开镜像窗口，然后返回车机桌面

### DISCUSSION THREAD

**Discussion by LiamJiang** [eiyooooo/Easycontrol_For_Car#12] [Ideas]
能不能加个选项：连接成功后自动打开镜像窗口，然后返回车机桌面，这样可以节省点时间。

**Comment by eiyooooo** [comment]
> 能不能加个选项：连接成功后自动打开镜像窗口，然后返回车机桌面，这样可以节省点时间。

设置设备为默认设备和设置里面打开打开默认后返回，打开软件自动连接之后会自动返回桌面

**Reply by LiamJiang** [reply]
是不是有线连接的设备不支持设置默认设备？没找到选项，有线连接成功后也不会自动返回桌面。建议实现：有线插上连接成功后自动打开手机镜像窗口，然后返回车机桌面。

然后我车机打开易控软件后会弹出【要允许易控访问null吗？】我觉得应该是车机还有其他USB设备？建议加个忽略某些USB设备选项？
![IMG_20240421_213635](https://github.com/eiyooooo/Easycontrol_For_Car/assets/44234300/3cd3f9a4-c26d-44d5-aee3-bbb3b980b3f7)

**Reply by eiyooooo** [reply]
> 是不是有线连接的设备不支持设置默认设备？没找到选项，有线连接成功后也不会自动返回桌面。建议实现：有线插上连接成功后自动打开手机镜像窗口，然后返回车机桌面。

部分车机支持USB自动授权则可以设置默认设备，否则需要手动授权无法默认打开，

> 然后我车机打开易控软件后会弹出【要允许易控访问null吗？】我觉得应该是车机还有其他USB设备？建议加个忽略某些USB设备选项？

必须要授权之后才能判断是否为adb设备，我过几天看看能不能加个忽略指定设备吧
